courses/machine_learning/deepdive2/structured/solutions/4c_keras_wide_and_deep_babyweight.ipynb

```diff
@@ -0,0 +1,814 @@
+{
```

Comment:
<p>similar comment here. i guess not great to hard code units=10 for 'wide_out' but the alternative would mean adding a variable such as 'wide_out_units' in the 'get_model_outputs' function. I think it's fine in this case, since these solutions are meant to illustrate a solution and not fully productionized code? Also, the usage of 'dnn_hidden_units' here is probably enough to show the alternative. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Yup, my thoughts exactly. More for teaching the main points and not worrying about extra things, at least not yet. </p>

Reply:
ok. sounds like we're on the same page.